mmdeploy/codebase/mmdet/deploy/object_detection.py

```diff
@@ -68,7 +68,7 @@ def process_model_config(model_cfg: Config,
         # for static exporting
         if input_shape is not None:
             if transform.type == 'Resize':
-                pipeline[i].keep_ratio = False
+                # pipeline[i].keep_ratio = False
```

Comment:
May recover it back since it may cause input shape not expected result.

Reply:
This part of hardcode will cause users to encounter potential bugs that are difficult to detect. Please fix this in the future or update the document.

Reply:
Yeah, we should have mentioned that in the document.